swimster-ui/src/hooks/useRegistrationForm.js

```diff
@@ -8,7 +8,6 @@ export const useRegistrationForm = () => {
     const { user, setUser } = useAuthContext()
     const navigate = useNavigate()
     const { register, handleSubmit } = useForm()
-    const [form, setForm] = useState({})
```

Comment:
It's not needed because the `react-hook-form` package automatically grabs those contents for me, like so

https://github.com/Facebook-University-Projects/Swimster/blob/dc7ce043b93924ae7279912559c0be682f521b5a/swimster-ui/src/components/Register/Register.jsx#L13-L19

The `handleSubmit` function is `react-hook-form`'s logic for getting all of the form data using `{...register(formName)}`